base_ebill_payment_contract/views/ebill_payment_contract.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2020 Camptocamp SA
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record model="ir.ui.view" id="ebill_payment_contract_form_view">
+        <field name="name">ebill.payment.contract.form</field>
+        <field name="model">ebill.payment.contract</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet>
+                    <group name="main">
+                        <field name="transmit_method_id" />
+                        <field name="state" />
+                        <field name="date_start" />
+                        <field name="date_end" />
+                        <field name="partner_id" />
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+    <record model="ir.ui.view" id="ebill_payment_contract_search_view">
+        <field name="name">ebill.payment.contract.search</field>
+        <field name="model">ebill.payment.contract</field>
+        <field name="arch" type="xml">
+            <search>
+                <field name="transmit_method_id" />
+                <field name="partner_id" />
+                <filter
+                    name="is_valid"
+                    string="Valid"
+                    domain="[('is_valid', '=', True)]"
+                />
+            </search>
+        </field>
+    </record>
+    <record model="ir.ui.view" id="ebill_payment_contract_tree_view">
+        <field name="name">ebill.payment.contract.tree</field>
+        <field name="model">ebill.payment.contract</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="transmit_method_id" />
+                <field name="partner_id" />
+                <field name="state" />
+                <field name="date_start" />
+                <field name="date_end" />
+                <field name="is_valid" invisible="1" />
+            </tree>
+        </field>
+    </record>
+    <record model="ir.actions.act_window" id="ebill_payment_contract_act_window">
+        <field name="name">Ebill Payment Contract</field>
+        <field name="res_model">ebill.payment.contract</field>
+        <field name="view_mode">tree,form</field>
+        <field name="domain">[]</field>
+        <field name="context">{}</field>
+    </record>
+    <record model="ir.ui.menu" id="ebill_payment_contract_menu">
+        <field name="name">Ebill Payment Contract</field>
```

Comment:
```suggestion
        <field name="name">eBill Payment Contract</field>
```

Reply:
Thanks for the search on terminology, I fixed them all

Reply:
Seems this line slipped out of the fixing

eBill is still written with a lowercase at the start of a sentence.

An other orthography in "E-bill" with the dash. But most people have seen it as "eBill".